win32ss/gdi/ntgdi/freetype.c

```diff
@@ -1365,16 +1971,16 @@ IntGdiLoadFontsFromMemory(PGDI_LOAD_FONT pLoadFont,
     {
         /* private font */
         PPROCESSINFO Win32Process = PsGetCurrentProcessWin32Process();
-        IntLockProcessPrivateFonts(Win32Process);
```

Comment:
Why is the per-process font lock dropped?

Reply:
If we use two locks, then what is the priority order of two locks?

Reply:
I'm caring about multiple mutex deadlock:

https://stackoverflow.com/questions/42720131/multiple-locks-with-mutex-and-the-possibility-of-a-deadlock